dashboard/cookbook/stagingdomain.md

```diff
@@ -1,19 +1,37 @@
+If the project URL is `example.com`, and a staging server exists at `dev.example.com`, you can enter that URL into the Staging Domain field and click on "Add Staging Domain". The domain is added to the menu below the text field automatically and enabled as default.
 
-Content translations are propagated like in all cases, but it important to keep in mind that *previous translations* will only show up appropriately if there is a 100% matching between the path structures of the original site and the staging server. That is, any entries translated on `/three/level/deep/path.html` will not be
+All requests going through the proxy (regardless of initiator, such as a user session in Preview or a content extraction Scan) will be mapped to that domain, regardless of the original project URL - the project domain might be `example.com`, but the Preview will be displaying content from `dev.example.com`.
 
-Hover over the staging domain name to reveal a menu with three options. *Live default*, *Default* and *Delete*. Click on "Make Live default" to enable the staging domain for the live published site. A tick icon will be displayed next to the staging domain with this option enabled. The same goes for *Default*, which applies to all other proxy modes (such as Highlight View in the Workbench, PReview or the X-proxy). Click on *Delete* to remove a Staging Domain.
+Content translations are propagated as usual, but this comes with an important warning: *previous translations* will only show up appropriately if the path structures of the original site and the staging server match 100%.
 
-Naturally, only one staging domain can be designated as Live or Preview default at one time.
+## Default and Live Default Staging domains
+
+Hover over the staging domain name to reveal three options. *Live default*, *Default* and *Delete*.
+
+![Staging Domain Example](/img/staging-domain.png)
+
+Click on "Make Live default" to enable the staging domain for the live published site. A tick icon will be displayed next to the staging domain when this option is enabled.
+
+The same goes for *Default*, which applies to all other proxy modes (such as the Highlight View in the Workbench, the Preview domain or the X-proxy). Click on *Delete* to remove a Staging Domain.
+
+Naturally, only one staging domain can be designated as Live or Preview default at one time, but one domain may be enabled for both
 
 ## When to Use Staging Domains?
 
 ### Content Decoupling
 
-### Domain Name Change
+The staging domain feature of the proxy is most beneficial when updates to the original site undergo a test-run on the staging server first. Staging domains let you make that same, preliminary content available through the proxy for translation work without disturbing the translation quality over the published domain in the meantime.
```

Comment:
"updates to the original site undergo a test-run" - ezt szerintem el kéne különíteni a bleeding elkerülésétől, mert a test run inkább a logic change-ekre vontakozik, szerintem